airbyte-integrations/bases/base-normalization/normalization/transform_catalog/transform.py

```diff
@@ -22,15 +22,270 @@
 SOFTWARE.
 """
 
+import argparse
+import json
 import os
-
-print(os)
+from typing import Optional, Tuple, Union
 
 
 class TransformCatalog:
-    def run(self):
-        print("running catalog transform")
+    """
+To run this transformation:
+```
+python3 main_dev_transform_catalog.py \
+  --catalog integration_tests/catalog.json \
+  --out normalization/dbt-transform/models/generated/ \
+  --json json_blob \
```

Comment:
my inclination would be to call this `--json-column`. that might be java speaking if you think it's more idiomatic in python to stick with one word that is less specific, than feel free to leave it as is.